dockerfiles/ci/buster/php-8.0/Dockerfile_shared_ext

```diff
@@ -62,6 +62,20 @@ COPY --chown=circleci:circleci --from=php-debug-zts-asan $PHP_INSTALL_DIR_DEBUG_
 COPY --chown=circleci:circleci --from=php-debug $PHP_INSTALL_DIR_DEBUG_NTS $PHP_INSTALL_DIR_DEBUG_NTS
 COPY --chown=circleci:circleci --from=php-nts $PHP_INSTALL_DIR_NTS $PHP_INSTALL_DIR_NTS
 
+# Build curl versions
+ENV CURL_VERSIONS="7.72.0 7.77.0"
+RUN set -eux; \
+    for curlVer in ${CURL_VERSIONS}; \
+    do \
+        echo "Build curl ${curlVer}..."; \
+        cd /tmp; \
+        curl -L -o curl.tar.gz https://curl.se/download/curl-${curlVer}.tar.gz; \
+        tar -xf curl.tar.gz && rm curl.tar.gz; \
+        cd curl-${curlVer}; \
+        ./configure --with-openssl --prefix=/usr/local/src/curl/${curlVer}; \
```

Comment:
Installing it to `/usr/local/src` seems weird. Shouldn't this be `/opt/curl/${curlVer}` or something?

Reply:
Nice catch. Fixed in 8853028.